src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -828,4 +844,15 @@ private FileStorageObject createFileStorageObject(String entityId, FileCategory
         );
         return fileStorageObjectDAO.findFileById(newFileStorageObjectId);
     }
+
+    private Dataset insertDataset() {
+        User user = createUser();
+        String name = "Name_" + RandomStringUtils.random(20, true, true);
+        Timestamp now = new Timestamp(new Date().getTime());
+        String objectId = "Object ID_" + RandomStringUtils.random(20, true, true);
+        DataUse dataUse = new DataUseBuilder().setGeneralUse(true).build();
+        Integer id = datasetDAO.insertDataset(name, now, user.getUserId(), objectId, true, dataUse.toString(), null);
+        createDatasetProperties(id);
+        return datasetDAO.findDatasetById(id);
+    }
```

Comment:
This is an intentional copy from `DAOTestHelper` - we're trying to move away from common methods in that class in favor of replicating the required behavior in specific DAO testing classes.